code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -82,24 +82,25 @@
 	..()
 	desc = initial(desc)
 	if(takes_damage)
-		var/servant_message = "It is at [health]/[max_health] integrity"
+		var/servant_message = "It is at <b>[health]/[max_health]</b> integrity"
 		var/other_message = "It seems pristine and undamaged"
 		var/heavily_damaged = FALSE
-		switch(health/max_health)
```

Comment:
I mean, i know it should have been `switch((health/max_health) * 100)` but that didn't actually work, for some reason? I've got no idea, honestly.